server/app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,48 @@
+class ReviewsController < ApplicationController
+  before_action :set_restaurant
+  before_action :set_review, only: %i(show update destroy)
+
+  # GET /restaurants/:restaurant_id/reviews
+  def index
+    @reviews = Review.where restaurant: @restaurant
```

Comment:
if you have `@restaurant`
the rails way would be to use the active record association:
`json_response @restaurant.reviews`

Reply:
Cool